lib/pages/new_group/new_group.dart

```diff
@@ -155,6 +169,102 @@ class NewGroupController extends State<NewGroup>
     );
   }
 
+  void uploadAvatarNewGroupChat({
+    required Client matrixClient,
+    required Uint8List file,
+    String? fileName,
+    String? contentType,
+  }) {
+    streamSubscription = uploadAvatarNewGroupChatInteractor.execute(
+      matrixClient: matrixClient,
+      file: file,
+      fileName: fileName,
+      contentType: contentType,
+    ).listen(
+      (event) => _handleUploadAvatarNewGroupChatOnData(context, event),
+      onDone: _handleUploadAvatarNewGroupChatOnDone,
+      onError: _handleUploadAvatarNewGroupChatOnError
+    );
+  }
+
+  void _handleUploadAvatarNewGroupChatOnData(BuildContext context, Either<Failure, Success> event) {
+    Logs().d('CreateRoomController::_handleUploadAvatarNewGroupChatOnData() - event: $event');
+    uploadAvatarNewGroupChatNotifier.value = event;
+  }
+
+  void _handleUploadAvatarNewGroupChatOnDone() {
+    Logs().d('CreateRoomController::_handleUploadAvatarNewGroupChatOnDone() - done');
+  }
+
+  void _handleUploadAvatarNewGroupChatOnError(dynamic error, StackTrace? stackTrace) {
+    Logs().e('CreateRoomController::_handleUploadAvatarNewGroupChatOnError() - error: $error | stackTrace: $stackTrace');
+  }
+
+
+  Future<AvatarAction?> _getSaveAvatarActions(BuildContext context) async {
+    final actions = [
+      if (PlatformInfos.isMobile)
+        SheetAction(
+          key: AvatarAction.camera,
+          label: L10n.of(context)!.openCamera,
+          isDefaultAction: true,
+          icon: Icons.camera_alt_outlined,
+        ),
+      SheetAction(
+        key: AvatarAction.file,
+        label: L10n.of(context)!.openGallery,
+        icon: Icons.photo_outlined,
+      ),
+    ];
+
+    final action = actions.length == 1
+        ? actions.single.key
+        : await showModalActionSheet<AvatarAction>(
```

Comment:
Why the `sheetAction` look like a widget of `iOS`. How to make it more M3 style ?